aemeds/scripts/scripts.js

```diff
@@ -448,7 +448,7 @@ function isSameDomainOrPdf(url) {
   return (
     !isPdf
     && (window.location.hostname === ancUrl.hostname
-      || ancUrl.hostname.toLowerCase().endsWith('servicenow.com')
+      || ancUrl.hostname.toLowerCase() === 'www.servicenow.com'
```

Comment:
let's also add a `|| ancUrl.hostname.toLowerCase() === 'servicenow.com'` , because I see they have a redirect from any `servicenow.com` -> `www.servicenow.com`, so we should open these too in the same tab is somebody adds them unintentionally.